.github/actions/javascript/proposalPoliceComment/action.yml

```
@@ -0,0 +1,15 @@
+name: 'ProposalPolice™ Comment Check Script'
+description: 'Verifies whether a proposal comment follows the template OR an updated proposal had substantial changes'
+inputs:
+  GITHUB_TOKEN:
+    description: 'Auth token for New Expensify Github'
+    required: true
+  OPENAI_API_KEY:
```

Let's update this to `PROPOSAL_POLICE_API_KEY`

```suggestion
  PROPOSAL_POLICE_API_KEY:
```